app/controllers/owners_controller.rb

```diff
@@ -10,19 +10,8 @@ def show
     # Only do this once
     session[:new_supporter] = false if @new_supporter
 
-    @scrapers = @owner.scrapers.includes(:last_run)
-
-    # Split out scrapers into different groups
-    @running_scrapers, @erroring_scrapers, @other_scrapers = [], [], []
-    @scrapers.each do |scraper|
-      if scraper.running?
-        @running_scrapers << scraper
-      elsif scraper.requires_attention?
-        @erroring_scrapers << scraper
-      else
-        @other_scrapers << scraper
-      end
-    end
+    @scrapers = @owner.scrapers.order(:updated_at).includes(:last_run)
+
```

Comment:
Nice, there's a bit of stray whitespace in some of the other changes than should be stripped :mag: